[ADD] new doc for release report (#1272)
mercadopago/devsite-docs: guides/manage-account/reports/available-money/api.pt.md

# Geração via API

> Você pode gerar seu relatório via API, tanto de forma manual quanto programada.

Reviewer: ```suggestion
```

> Ganhe tempo e automatize a **frequência de geração** do relatório de Dinheiro disponível sempre que quiser.

Reviewer: ```suggestion
Gere manualmente o relatório de dinheiro liberado quantas vezes quiser ou programe-o de acordo com a frequência que você escolher por meio de nossa API.```

>WARNING
>

> > O relatório de Dinheiro disponível será desabilitado a partir de 1o de março de 2022.

Reviewer: ```suggestion
> O relatório de Dinheiro disponível será desabilitado a partir de 1° de março de 2022.
```

>
> Você pode usar o [relatório de ----[mla]----Liquidações------------ ----[mlm, mlb, mlc, mco, mlu, mpe]----Liberações------------](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/manage-account/reports/released-money/introduction) para fazer a reconciliação das transações que afetem o saldo disponível na sua conta, incluindo seus saques bancários.

## Configurar seus relatórios

> Execute o curl que você precisa para gerar, consultar e atualizar seus relatórios.

Reviewer: ```suggestion
Você pode configurar seus relatórios de acordo com o que você precisa. A seguir, mencionamos quais são as chamadas de API que você pode fazer para criar, consultar e atualizar seus relatórios.
```

HttpsURLConnection connection = (HttpsURLConnection) url.openConnection();

connection.setRequestMethod("GET");
connection.setRequestProperty("Accept", "application/json");
connection.setRequestProperty("Content-Type", "application/json");
connection.setRequestProperty("Authorization", "Bearer ENV_ACCESS_TOKEN");

System.out.println(connection.getResponseCode());
System.out.println(connection.getResponseMessage());
System.out.println(connection.getInputStream());
```
```python
import requests
headers = {
    'accept': 'application/json',
    'content-type': 'application/json',
    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
}

response = requests.get('https://api.mercadopago.com/v1/account/bank_report/config', headers=headers)
```
```node
var request = require('request');

var headers = {
    'accept': 'application/json',
    'content-type': 'application/json',
    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
};

var options = {
    url: 'https://api.mercadopago.com/v1/account/bank_report/config',
    headers: headers
};
function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}
request(options, callback);
```
]]]

> Como resposta, você receberá um `HTTP STATUS 200 (Ok)`

Reviewer: ```suggestion
Como resposta, você receberá um `HTTP STATUS 200 (Ok)`:
```

connection.setRequestMethod("POST");
connection.setRequestProperty("Accept", "application/json");
connection.setRequestProperty("Content-Type", "application/json");
connection.setRequestProperty("Authorization", "Bearer ENV_ACCESS_TOKEN");

System.out.println(connection.getResponseCode());
System.out.println(connection.getResponseMessage());
System.out.println(connection.getInputStream());
```
```python
import requests

headers = {
    'accept': 'application/json',
    'content-type': 'application/json',
    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
}

response = requests.post('https://api.mercadopago.com/v1/account/bank_report/schedule', headers=headers)
```
```node
var request = require('request');

var headers = {
    'accept': 'application/json',
    'content-type': 'application/json',
    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
};

var options = {
    url: 'https://api.mercadopago.com/v1/account/bank_report/schedule',
    method: 'POST',
    headers: headers
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);
```
]]]

> Como resposta, você receberá um `HTTP STATUS 200 (Ok)`

Reviewer: ```suggestion
Como resposta, você receberá `HTTP STATUS 200 (Ok)`:
```

```json
{
    "id": 2541818,

>     "user_id": "USER-ID",

Reviewer: ```suggestion
    "user_id": "USER_ID",
```
